paddle/fluid/pybind/op_function_generator.cc

```diff
@@ -130,10 +130,25 @@ std::map<std::string, std::set<std::string>> op_passing_outs_map = {
     {"moving_average_abs_max_scale", {"OutScale", "OutAccum", "OutState"}},
 };
 
+// NOTE(pangyoki): Reuse Allocation Inplace Op.
```

Comment:
`Tensor view` is more explicit than `Reuse Allocation Inplace Op`

Reply:
done